src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/IHost.cs

```diff
@@ -13,12 +13,13 @@ namespace Microsoft.Extensions.Hosting
     public interface IHost : IDisposable
     {
         /// <summary>
-        /// The programs configured services.
+        /// The services configured for the program (e.g. using <see cref="M:HostBuilder.ConfigureServices(Action<HostBuilderContext,IServiceCollection>)" />).
         /// </summary>
         IServiceProvider Services { get; }
 
         /// <summary>
-        /// Start the program.
+        /// Start the <see cref="P:IHost.Services" /> configured for the program.
```

Comment:
```suggestion
        /// Start the <see cref="IHostedService" /> objects configured for the program.
```

I wouldn't reference the `Services` property here. Those are all the DI services for the program. They don't get started by this. Instead, the `IHostedService` instances do.